### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by johnny-vulture** [miskatonicstudio/goat#299]
not really an issue but i would request better docs to help intergrate the goat into our project

**Comment by miskatonicstudio** [comment]
Hi and thanks for reaching out! I agree that the quality of documentation could be greatly improved in GOAT :) The official one on [readthedocs](https://miskatonicstudio-goat.readthedocs.io/) is a bit outdated right now, with several new features being implemented recently and not mentioned there.

There are several issues already created for this: #27 , #223 , #243, #264. Ideally, not only the documentation would be up-to-date, but I'd like to create some kind of video tutorials, to make it easier for developers to start using GOAT.

However, being the only person developing GOAT, I need to prioritize other tasks right now :( GOAT was undergoing some changes recently, with a new direction I wanted to take: make this a project for launching multiple interactive games, preferably "escape room" type of games. However, I realized there are some limitations to this approach and I'm now considering different ones. I'd like to first clarify the goal of GOAT, then refactor some features it already has and add new ones. Once this is done, I'd like to update the docs.

I know that this might take a while, and I'm sorry if it impedes integrating GOAT with your project. If you have any questions or problems, please don't hesitate to ask them here (or create a separate issue with a description) and I'll answer ;)
